docs/sources/enterprise/access-control/plan-rbac-rollout-strategy.md

```diff
@@ -148,28 +148,96 @@ curl --location --request POST '<grafana_url>/api/access-control/roles/' \
 
-- Assign the `fixed:reporting:admin:edit` role to the `Viewer` basic role. For more information about assigning a fixed role to a basic role, refer to [Assign a fixed role to a basic role using provisioning]({{< relref "./assign-rbac-roles#assign-a-fixed-role-to-a-basic-role-using-provisioning" >}}).
+- Add the `fixed:reports:writer` role permissions to the `basic:viewer` role using the `role > from` list of your provisioning file:
 
-  > **Note:** The `fixed:reporting:admin:edit` role assigns more permissions than just creating reports. For more information about fixed role permission assignments, refer to [Fixed role definitions]({{< relref "./rbac-fixed-basic-role-definitions#fixed-role-definitions" >}}).
+```yaml
+apiVersion: 2
 
-- [Create a custom role]({{< ref "./manage-rbac-roles#create-custom-roles" >}}) that includes the `reports.admin:write` permission, and add the custom role to the `Viewer` basic role.
-  - For more information about assigning a custom role to a basic role, refer to [Assign a custom role to a basic role using provisioning]({{< relref "./assign-rbac-roles#assign-a-custom-role-to-a-basic-role-using-provisioning" >}}) or [Assign a custom role to a basic role using the HTTP API]({{< relref "./assign-rbac-roles#assign-a-custom-role-to-a-basic-role-using-the-http-api" >}}).
+roles:
+  - name: 'basic:viewer'
+    global: true
+    version: 3
+    from:
+      - name: 'basic:viewer'
+        global: true
+      - name: 'fixed:reports:writer'
+        global: true
+```
+
+> **Note:** The `fixed:reports:writer` role assigns more permissions than just creating reports. For more information about fixed role permission assignments, refer to [Fixed role definitions]({{< relref "./rbac-fixed-basic-role-definitions#fixed-role-definitions" >}}).
+
+- Or add the following permissions to the `basic:viewer` role, using provisioning or the [RBAC HTTP API]({{< relref "../../http_api/access_control.md#update-a-role" >}}):
+
+| action                 | scope                           |
+| ---------------------- | ------------------------------- |
+| `reports.admin:create` | n/a                             |
+| `reports.admin:write`  | `reports:*` <br> `reports:id:*` |
+| `reports:read`         | `reports:*`                     |
+| `reports:send`         | `reports:*`                     |
 
 ### Prevent a Grafana Admin from creating and inviting users
 
-This topic describes how to remove the `users:create` permissions from the Grafana Admin role, which prevents the Grafana Admin from creating users and inviting them to join an organization.
+To prevent a Grafana Admin from creating users and inviting them to join an organization. You have to [update a basic role permissions]({{< ref "./manage-rbac-roles.md#update-a-basic-role-permissions" >}}).
```

Comment:
```suggestion
To prevent a Grafana Admin from creating users and inviting them to join an organization [update a basic role permissions]({{< ref "./manage-rbac-roles.md#update-a-basic-role-permissions" >}}).
```